src/components/Nav/Admin/Admin.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { IoMdClose } from 'react-icons/io';
+
+// COMPONENTS
+import UserFormInputs from '../UserInput/UserInput';
+import UserFormBtns from '../UserButton/UserFormBtn';
+import UserRadioBtns from '../UserRadioBtns/UserFormRadio';
+
+// STYLES
+import * as S from './AdminEle';
+
+// import { BASE_URL } from '../../../config';
+
+function Admin({ form, modalClose }) {
+  const { Kakao } = window;
+  const [radio, setRadio] = useState('');
+  const [inputValues, setInputValues] = useState({
+    name: '',
+    email: '',
+    password: '',
+  });
+
+  const handleSubmit = e => {
+    e.preventDefault();
+  };
+
+  const getInputValue = e => {
+    const { name, value } = e.target;
+
+    setInputValues({ ...inputValues, [name]: value });
+  };
```

Comment:
이전 값을 기반으로 수정하는 거라면 가급적 외부 값에 의존하지 않는 함수 업데이트 형태가 바람직합니다
```suggestion
  const getInputValue = e => {
    const { name, value } = e.target;

    setInputValues(prev => ({ ...prev, [name]: value }));
  };
```

Reply:
> 이전 값을 기반으로 수정하는 거라면 가급적 외부 값에 의존하지 않는 함수 업데이트 형태가 바람직합니다

copy를 만들어서 사용하는게 좋다는 말씀이신가요?????